sentry-core/src/main/java/io/sentry/core/UncaughtExceptionHandlerIntegration.java

```diff
@@ -82,7 +82,7 @@ static Throwable getUnhandledThrowable(Thread thread, Throwable thrown) {
     Mechanism mechanism = new Mechanism();
     mechanism.setHandled(false);
     mechanism.setType("UncaughtExceptionHandler");
-    return new ExceptionMechanismThrowable(mechanism, thrown, thread);
+    return new ExceptionMechanism(mechanism, thrown, thread);
```

Comment:
This part became confusing

https://docs.sentry.io/development/sdk-dev/event-payloads/exception/#exception-mechanism